src/README.md

```diff
@@ -0,0 +1,45 @@
+# Source Structure
```

Comment:
Maybe when we are all starting to work on the dashboard but it would be good to do an overview of this architecture with the group. It could also be useful for our low level design!

Reply:
After what we discussed in our meeting on Thursday, should we still have this in here? I forget if we decided if it's okay to have different structures in the Vue app